spec/pdc/v1/global_component_contact_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe PDC do
+  before do
+    VCR.insert_cassette fixture_name
+  end
+
+  after do
+    VCR.eject_cassette
+  end
+
+  let(:contact) { PDC::V1::GlobalComponentContact }
+
+  it 'must has compose' do
```

Comment:
is this "must have component"

Reply:
component contact doesn't have compose,
should be "must have component contact", and at meanwhile, please check the vcr file name accordingly.